Frameworks/UIKit/UIWebView.mm

```diff
@@ -320,7 +331,7 @@ - (void)dealloc {
         [_xamlWebControl removeLoadCompletedEvent:_xamlLoadCompletedEventCookie];
         [_xamlWebControl removeNavigationStartingEvent:_xamlLoadStartedEventCookie];
         [_xamlWebControl removeUnsupportedUriSchemeIdentifiedEvent:_xamlUnsupportedUriSchemeEventCookie];
-        [_xamlWebControl release];
+        _xamlWebControl = nil;
```

Comment:
not require usually, but here we're freeing the element on the UIThread.

---
In reply to: [86633009](https://github.com/Microsoft/WinObjC/pull/1318#discussion_r86633009) [](ancestors = 86633009)